ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue

```diff
@@ -16,34 +16,142 @@
+                </v-col>
+                <v-col>
+                  <p class="first-col pt-1 font-weight-bold">
+                    {{ getName() }}
+                  </p>
+                </v-col>
+              </v-row>
+            </v-col>
+            <v-col cols="3" class="pl-1">
+              <base-address
+                v-if="!emptyAddress()"
+                class="content"
+                :schema="addressSchema"
+                :value="address"
+              >
+              </base-address>
+              <p v-else class="content"> (Not Entered) </p>
+            </v-col>
+            <v-col cols="3" class="pl-3">
+              <p class="content">{{getMhrRegistrationSubmittingParty.emailAddress || '(Not Entered)'}}</p>
+            </v-col>
+            <v-col cols="3" class="pl-4">
+              <p class="content" v-html="parsePhoneNumber()"></p>
+            </v-col>
+          </v-row>
+
+          <div class="px-4">
+            <v-divider />
+          </div>
+
+          <v-row no-gutters class="px-6 py-7">
+            <v-col cols="3">
+              <p class="first-col">Attention or<br>Reference Number</p>
+            </v-col>
+            <v-col cols="9">
+              <p class="content">{{getMhrAttentionReferenceNum || '(Not Entered)'}}</p>
+            </v-col>
+          </v-row>
         </section>
       </template>
     </div>
   </v-card>
 </template>
 
 <script lang="ts">
-import { defineComponent, reactive, toRefs } from '@vue/composition-api'
+import { computed, defineComponent, reactive, toRefs } from '@vue/composition-api'
 import { RouteNames } from '@/enums'
+import { useGetters } from 'vuex-composition-helpers'
+import { BaseAddress } from '@/composables/address'
+import { PartyAddressSchema } from '@/schemas'
 
 export default defineComponent({
   name: 'SubmittingPartyReview',
-  components: {},
+  components: {
+    BaseAddress
+  },
   props: {},
   setup () {
-    const localState = reactive({})
+    const {
+      getMhrRegistrationSubmittingParty,
+      getMhrAttentionReferenceNum
+    } = useGetters<any>([
+      'getMhrRegistrationSubmittingParty',
+      'getMhrAttentionReferenceNum'
+    ])
+
+    const localState = reactive({
+      address: computed(() => getMhrRegistrationSubmittingParty.value.address),
+      businessName: computed(() => getMhrRegistrationSubmittingParty.value.businessName),
+      personName: computed(() => getMhrRegistrationSubmittingParty.value.personName)
+    })
+
+    const addressSchema = PartyAddressSchema
+
+    const parsePhoneNumber = () => {
+      const phone = getMhrRegistrationSubmittingParty.value
+      const phoneNum = phone.phoneNumber
+      const ext = phone.phoneExtension ? ' &nbsp;Ext ' + phone.phoneExtension : ''
+      if (!phoneNum) {
+        return '(Not Entered)'
+      }
+      return `${phoneNum}${ext}`
+    }
+
+    const emptyAddress = () => {
+      return Object.values(localState.address).every(val => !val)
+    }
```

Comment:
As we discussed, this would be better suited as a computed property. 
Returning a singular value type without any arguments or potential for error responses, makes sense as a getter.

Reply:
Also, might be better to update to `hasAddress` as true values are a little easier to read than a `!emptyAddress`